sdk/spring/azure-spring-boot-core/pom.xml

```diff
@@ -11,12 +11,12 @@
   </parent>
 
   <groupId>com.azure.spring</groupId>
-  <artifactId>azure-identity-spring</artifactId>
-  <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-identity-spring;current} -->
+  <artifactId>azure-spring-boot-core</artifactId>
+  <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot-core;current} -->
   <packaging>jar</packaging>
 
-  <name>Azure Identity Spring Integration Library</name>
-  <description>Azure Identity Spring Integration Library</description>
+  <name>Azure Core Spring Integration Library</name>
+  <description>Azure Core Spring Integration Library</description>
```

Comment:
The description also needs to be changed.

Reply:
yeah, I like this idea, "Azure Core Spring Integration Library" really confusing people, and we really should get rid of that "integration" from most of our lib , unless the ones really doing integration work